app/templates/index.html

```diff
@@ -0,0 +1,19 @@
+<html>
```

Comment:
I figured having a templating engine would be good for serving error pages. It was also practice with golang.